charter.md

```diff
@@ -4,6 +4,8 @@ The More Instant Messaging Interoperability (MIMI) working group will specify th
 
 Recognizing the need for a standardized security protocol to support group key establishment, authentication, and confidentiality services for messaging, the IETF has specified the Messaging Layer Security (MLS) protocol [I-D.ietf-mls-protocol] and architecture [I-D.ietf-mls-architecture]. MLS is agnostic to the identity system used within any given messaging service; it provides confidentiality of sessions once the participants in a conversation have been identified. To achieve interoperable messaging, the MIMI working group will specify how one or more identity building block technologies (for example, X.509 certificates or Verifiable Credentials) can be used to establish end-to-end cryptographic identity across messaging services, assuming the use of MLS for key establishment. 
 
+Interoperable messaging in federated environments requires consensus on a common delivery service and a transport protocol between federated domains. Specifically, the MLS protocol requires ordering of handshake messages within groups to ensure clients can synchronize despite asynchronous message delivery. This working group will specify a flexible solution that takes into account typical requirements and best practices from the industry. 
```

Comment:
This feels like it's excluding (minor) modifications to MLS as a solution to better support federated/decentralized environments, though overall the importance of supporting an MLS-capable federated protocol is very welcome.

Reply:
Extensions to MLS (which is how modifications would probably be implemented) are listed as out-of-scope a few lines below. The WG could come up with additional requirements, though, which the MLS WG (or anyone else) could then draft an extension for. Not sure if we need to mention this in the charter, though.